java/change-notes/2021-06-28-unsafe-cert-trust-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The query "Unsafe certificate trust" (`java/unsafe-cert-trust`) has been promoted from experimental to the main query pack. Its results will now appear by default. This query was originally [submitted as an experimental query by @luchua-bc](https://github.com/github/codeql/pull/3550).
```

Comment:
Needs conversion to new change note format.

Reply:
Done by rebasing.